src/main/java/br/com/sbs/cubatech/category/CategoryRepository.java

```diff
@@ -0,0 +1,16 @@
+package br.com.sbs.cubatech.category;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+
+    @Query(value = "SELECT c FROM Category c WHERE c.status = 'ACTIVE' ")
+    List<Category> findByStatusActive();
```

Comment:
Daria pra fazer assim:

```suggestion
    List<Category> findByStatus(Status status);
```

Aí tem que mudar em `CategoryApiController`.